tests/v2/validation/nodescaling/README.md

```diff
@@ -21,50 +21,42 @@ rancher:
 ```
 
 ## Node Replacing
-Node replacement tests require that the given pools have unique, distinct roles and more than 1 node per pool. You can run a subset of the tests, but still need more than 1 node for the role you would like to run the test for. i.e. for `-run ^TestScaleDownAndUp/TestWorkerScaleDownAndUp$` you would need at least 1 pool with 2 or more dedicaated workers in it. The last node in the pool will be replaced. 
-
-Typically, a cluster with the following 3 pools is used for testing:
+Node replacement tests require that the given pools have unique, distinct roles and more than 1 node per pool. Typically, a cluster with the following 3 pools is used for testing:
 ```yaml
-{
-  {
-    ControlPlane: true,
-    Quantity:     2,
-  },
-  {
-    Etcd:     true,
-    Quantity: 3,
-  },
-  {
-    Worker:   true,
-    Quantity: 2,
-  },
-}
+provisioningInput:
+  nodePools:
```

Comment:
since it depends if you're running rke2 or rke1, is there a way to specify that for the user? i.e. nodePools == rke1, machinePools == rke2/k3s?

Reply:
Resolved.